pkg/services/fileerror/file_error.go

```diff
@@ -1,4 +1,4 @@
-package services
+package fileerror
```

Comment:
I think it's better if we merge `pkg/services/fileerror` into `pkg/services/file`. What you think?